omnigibson/simulator.py

```diff
@@ -489,53 +594,36 @@ def import_scene(self, scene):
             assert self.is_stopped(), "Simulator must be stopped while importing a scene!"
             assert isinstance(scene, Scene), "import_scene can only be called with Scene"
 
-            # Clear the existing scene if any
-            self.clear()
-
-            # Initialize all global updatable object states
-            for state in self.object_state_types_requiring_update:
-                if issubclass(state, GlobalUpdateStateMixin):
-                    state.global_initialize()
+            # Check that the scene is not already imported
+            if scene in self._scenes:
+                raise ValueError("Scene is already imported!")
 
-            self._scene = scene
-            self._scene.load()
+            # Load the scene.
+            self._scenes.append(scene)
+            # TODO(parallel-hang): Actually implement this API
+            self._last_scene_edge = scene.load(idx=len(self.scenes), last_scene_edge=self._last_scene_edge)
```

Comment:
what is this exactly?

Reply:
I'll fix this real soon, you can check this comment from Eric: https://github.com/StanfordVL/OmniGibson/pull/699#discussion_r1630247157. 
Essentially, we are going to have the simulator keep track of these "last scene edge" for scene positioning instead of the scenes themselves.